source/services/encryption/EncryptionService.stories.tsx

```diff
@@ -110,9 +117,28 @@ const testSymmetricKeySetup = async () => {
   printPublicKeyResult(coApplicantStina, updatedForm);
   await printSymmetricKeyResult(coApplicantStina, updatedForm);
 
-  await setupSymmetricKey(mainApplicantYlva, updatedForm);
+  updatedForm = await setupSymmetricKey(mainApplicantYlva, updatedForm);
   await printSymmetricKeyResult(mainApplicantYlva, updatedForm);
 
+  updatedForm = await encryptFormAnswers(mainApplicantYlva, updatedForm);
+  const encryptedFormCopy = JSON.parse(JSON.stringify(updatedForm));
+
+  const { answers: decryptedAnswers1 } = await decryptFormAnswers(
+    mainApplicantYlva,
+    updatedForm
+  );
+
+  const updatedFormCopy = JSON.parse(JSON.stringify(encryptedFormCopy));
+  const { answers: decryptedAnswers2 } = await decryptFormAnswers(
+    coApplicantStina,
+    updatedFormCopy
+  );
+
+  console.log("encrypted", encryptedFormCopy.answers);
+  console.log("encrypted type:", encryptedFormCopy.encryption.type);
+  console.log("decrypted (applicant)", decryptedAnswers1);
+  console.log("decrypted (co-applicant)", decryptedAnswers2);
+
   console.log("Cleaning up...");
   console.log("Removing any stored symmetric keys.");
```

Comment:
Are these logs intentional?

Reply:
The first logs are intentional as that is what we want to verify when running the stories tests. The "Cleaning up..." one is redundant though I think; I'll clean it up 🙂.